random/resource_integer_test.go

```diff
@@ -20,6 +20,12 @@ func TestAccResourceIntegerBasic(t *testing.T) {
 					testAccResourceIntegerBasic("random_integer.integer_1"),
 				),
 			},
+			{
+				ResourceName:      "random_integer.integer_1",
+				ImportState:       true,
+				ImportStateId:     "3,1,3,12345",
+				ImportStateVerify: true,
+			},
```

Comment:
Ideally, we'd see a version of this that didn't set the Seed, just to exercise that code path, too.

Reply:
That's a difficult test to write: the import tests create a resource and then import it and check that the result matches. How would you go about guaranteeing that the resource will be created in that first step with a particular result without the seed?

Reply:
That's a fair question! Request rescinded.